views/setting/index.php

```diff
@@ -22,6 +22,8 @@
         <?php if($settings->hasGallery()) : ?>
             <?php $form = ActiveForm::begin() ?>
                 <?= $form->field($settings, 'snippetGallery')->dropDownList($settings->getGallerySelection())?>
+                <?= $form->field($settings, 'sortOrder')->textInput(['type' => 'number']) ?>
+<!--        'min' =>  $settings::SORT_MIN, 'max' => $settings::SORT_MAX, 'placeholder' => Yii::t('GalleryModule.base', 'from ' . $settings::SORT_MIN . ' to ' . $settings::SORT_MAX . ' in increments of 100'), 'maxLength' => 4]-->
```

Comment:
Remove

Reply:
Do you mean "Remove" the commented out line number 26, isn't it?

Thanks!

Reply:
Yes, just remove Line 26 which is commented out anyway.

Reply:
I have already pushed a few commits (I believe you can see it in my old PR).
I leave it like this:
`<?= $form->field($settings, 'sortOrder')->textInput(['type' => 'number', 'placeholder' => Yii::t('GalleryModule.base', 'from ' . $settings::SORT_MIN . ' to ' . $settings::SORT_MAX . ' in increments of 100'),]) ?>`

Please, let me know if it what you wanted.
Thank you for your quick reply!